network/manifest-1/README.md

```diff
@@ -45,17 +54,27 @@ git checkout VERSION
 
 make install # go install ./...
 # For ledger support `go install -tags ledger ./...`
-
 manifestd config set client chain-id manifest-1
 ```
 
+OR
+
+```bash
+wget <link to manifest precompile>
+chmod +x manifestd
+mv manifestd /usr/local/bin
+```
+
```

Comment:
Refer to this section in POST_GENESIS?

Reply:
this is apart of the install steps. Both pre and post genesis. So it should be done before pre-genesis